.github/workflows/dotnet-format.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       run: dotnet tool install -g dotnet-format
```

Comment:
If it's part of the newer .NET SDK, then do we still need to install `dotnet-format`?